src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int figures = 5;
+    private static final int maxSize = 99;
+    private static final String defaultColor = "white";
+    private static final int defaultRadius = 10;
+
+    public Figure getRandomFigure() {
+        return getFigure();
+    }
+
+    private Figure getFigure() {
+        final int figures = 5;
+        final int maxSize = 99;
+        ColorSuplier colorSuplier = new ColorSuplier();
+        int numberFigure = random.nextInt(figures);
+        String color = colorSuplier.getRandomColor();
+        switch (numberFigure) {
+            case 1:
+                return getRandomCircle(maxSize, color);
```

Comment:
actually you can simplify your solution
```suggestion
                return new Circle(defaultColor, defaultRadius);
```
the same for other figures, as you have constructors with parameters and you will create 
```
public int randomSize() {
     return random.nextInt() * maxSize;
}
```

Reply:
It isn't good idea. I done the same. But ser Okuzan advised me the next.[okuzan](https://github.com/okuzan) [13 hours ago](https://github.com/mate-academy/jv-oop-advanced/pull/1589#discussion_r1597047234)
for each of these cases, create a private method (createRandomRightTriangle etc), this method becomes too big and unwieldy/. And after it looks like better.

Reply:
+